other/webpack.config.es6.js

```diff
@@ -40,7 +40,6 @@ const pkg = require('../package.json');
 const bannerText = fs.readFileSync(path.resolve(__dirname, '../BANNER.txt')).toString();
 
 const config = {
-  cache: false,
```

Comment:
When does the cache bust? We wouldn't run into issues with changes not showing up, right?